lib/nacre/concerns/getable.rb

```diff
@@ -13,7 +13,7 @@ def get(range)
   private
 
   def url
-    service_url + '/' + service_name
+    service_url + '/' + resource_name
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.